### PR TITLE
fix(aws): handle none type attributes

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Handle `None` type attributes in checks:
- `organizations_scp_check_deny_regions`
- `ecr_repositories_not_publicly_accessible`

Also, fix typo in work `accessible`.

### Checklist

- Are there new checks included in this PR? Yes / No
    - If so, do we need to update permissions for the provider? Please review this carefully.
- [x] Review if the code is being covered by tests.
- [x] Review if code is being documented following this specification https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
- [x] Review if backport is needed.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/providers/aws/services/organizations/organizations_scp_check_deny_regions/organizations_scp_check_deny_regions.py
Comment (jfagoagas) [inline prowler/providers/aws/services/organizations/organizations_scp_check_deny_regions/organizations_scp_check_deny_regions.py:1]: Could you please add tests for the following lines:
- 74
- 82 to 85
- 88 to 90

If we are changing the logic of the check we should cover those scenarios.

**Review by jfagoagas — CHANGES_REQUESTED** [review]

File: prowler/providers/aws/services/organizations/organizations_scp_check_deny_regions/organizations_scp_check_deny_regions.py
Comment (MrCloudSec) [reply prowler/providers/aws/services/organizations/organizations_scp_check_deny_regions/organizations_scp_check_deny_regions.py:1]: @jfagoagas the logic of the check was not changed, only the following line was added:
`if org.policies is not None: # Access denied to list policies`

Comment (jfagoagas) [reply prowler/providers/aws/services/organizations/organizations_scp_check_deny_regions/organizations_scp_check_deny_regions.py:1]: Yes but even with that I think those lines are not being tested, you should add a test to verify when `org.policies is None`, don't you?

Comment (MrCloudSec) [reply prowler/providers/aws/services/organizations/organizations_scp_check_deny_regions/organizations_scp_check_deny_regions.py:1]: Yes, test was added!

**Review by jfagoagas — APPROVED** [review]
Great one, thanks @sergargar 🚀

**Pull request merged**